content/blog/2018/12/2018-12-14-java11-preview-availability.adoc

```diff
@@ -14,6 +14,11 @@ links:
   sig: platform
 ---
 
+[WARNING]
+--
+Please refer to link:/doc/administration/requirements/jenkins-on-java-11[Running Jenkins on Java 11] documentation to have the up-to-date details on how to run Jenkins on Java 11.
+--
+
```

Comment:
Suggest "Please refer to the link:/doc/administration/requirements/jenkins-on-java-11[Running Jenkins on Java 11] documentation for up-to-date details on running Jenkins on Java 11."